var/spack/repos/builtin/packages/py-pyspoa/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyPyspoa(PythonPackage):
+    """Python bindings to spoa"""
+
+    homepage = "https://github.com/nanoporetech/pyspoa"
+    pypi = "pyspoa/pyspoa-0.0.8.tar.gz"
+
+    version("0.0.8", sha256="8299d18066b498a6ef294c5a33a99266ded06eeb022f67488d2caecba974b0a4")
+
+    depends_on("py-wheel@0.34.1:", type="build")
+    depends_on("py-setuptools", type="build")
+    depends_on("cmake@3.18.4", type="build")
+    depends_on("py-pybind11@2.4:", type=("build", "run"))
+    depends_on("py-auditwheel@3.1.0:", type=("build", "run"))
+    depends_on("py-scikit-build@0.11.1", type=("build", "run"))
```

Comment:
auditwheel and scikit-build aren't used by this project

Reply:
yeah, I was looking at the "Developer Quick Start" where it tells you to run `pip install -r requirements.txt`. Digging through the source code, it looks like auditwheel is [only used in this script](https://github.com/nanoporetech/pyspoa/blob/v0.0.8/.github/workflows/build-wheels.sh), and scikit-build isn't used at all. I'll try re-building with those removed. Odd that the devs included it.

Reply:
`requirements.txt` is used by devs to install the libraries needed to use the software but also the development tools needed to maintain it. You should never check `requirements.txt` unless `setup.py` explicitly uses it in some way.